### PR TITLE
Test task

### PR DESCRIPTION
**Pull request by ErmakovDenis**
Hi all! Thanks for the left comments, I tried to fix them all, but not everything worked out, let's understand:
I started by walking on all 'decorative' editors (corrected the names, removed the excess of the spaces)
Let's talk about Dockerfile, made it shorter, removed the first stage, now /Dist is going to CI, corrected the port. I completely agree with Dmitry regarding User, I checked that the container is not starting from under Root, nevertheless, I didn’t succeed in directly indicating it - I get mistakes, I hope you tell me what the reason may be.

I corrected nginx.conf - now it is safer, I hope I understood the comment correctly.

Probably the most offensive that ci.yml all the time falls into a mistake, the main reason is the tests, they fall by default. If we ignore this step, it would seem that everything should work, but not quite.
At the Build and Push Docker Image step, I get an error about the lack of rights to recording the container, and the reason is as follows, I use Github_token which works only in the Master branch, there is a bypass using the CR_PAT key, but it fundamentally does not want to work tremendously, despite the time that I get Success messages (I am attaching screenshots).
image
image

We’ll talk about hardcodding, in some places I consider it appropriate (in this work), so Dmitry wrote to take the Node.js version from the Package.json file, but in this repository the developer did not indicate it (do not need to be confused with Version, the nods versions are not recorded In such a format as here 18.2.1) (correctly 18.20.1)
For the examiner, I changed the value of the variable to 18.20.1 and pulled this value in CI.YML.

I also could not find how to use GH CLI, except that I only use additional extensions. (correct me).
Messages to the mail come correctly, I also attach screenshots.
image
image

I hope I have not forgotten anything.

### CODE REVIEW

**Comment by kseniyashaydurova** [comment]
@ErmakovDenis pls close tis PR and reopen the previous one - https://github.com/ErmakovDenis/angular-react-starter/pull/1 Because now it is hard to compare the differences you did and the stuff we requested among multiple PRs. Now we can't see our comments and need to open 2 PRs and recheck each comment fix independently + we need to review all files again (if you did like I explained below, we would be able to check only new changes you did, not everything again).

So basically the workflow is: **if we reject your PR -> you make all required changes as new commits to the same branch -> when you are ready with all the changes, you rerequest PR check from me and Dmitry in the same PR**

**Comment by kseniyashaydurova** [comment]
@ErmakovDenis related to your issues and errors, pls spend more time to try to resolve issues yourself. We can tell you the answers, but the test task assumes that you should solve problems yourself or at least spend more time on trying to achive that. 

If you fixed all comments and script works ok, then you rerequest PR, also it would be much more convenient if you added you thoughts and comments related to current implementation in exact code lines.

**Review by kseniyashaydurova — CHANGES_REQUESTED** [review]
Pls add these changes to your previous PR
